src/Microsoft.DotNet.Cli.Utils/EnvironmentProvider.cs

```diff
@@ -54,6 +56,19 @@ private IEnumerable<string> SearchPaths
             }
         }
 
+        private string ExpandTildeSlash(string path)
+        {
+            const string tildeSlash = "~/";
+            if (path.StartsWith(tildeSlash) && !string.IsNullOrEmpty(_userHomeDirectory.Value))
```

Comment:
See https://github.com/dotnet/cli/pull/8589#discussion_r167697105 (outdated now)

Reply:
I changed to `StringComparison.Ordinal`. Is that change correct?

Reply:
Yes. Approved.